set.mm

```diff
@@ -437141,9 +437141,12 @@ orthogonal vectors (i.e. whose inner product is 0) is the sum of the
 htmldef "EqvRel" as ' EqvRel ';
   althtmldef "EqvRel" as ' EqvRel ';
   latexdef "EqvRel" as "\mathrm{EqvRel}";
-htmldef "ElEqvRel" as ' ElEqvRel ';
-  althtmldef "ElEqvRel" as ' ElEqvRel ';
-  latexdef "ElEqvRel" as "\mathrm{ElEqvRel}";
+htmldef "CoElEqvRels" as ' CoElEqvRels ';
```

Comment:
This affects the global token space.  In future someone might want to reserve the same tokens for a different purpose.  In order to allow parallel development not interfering each other, I suggest to add a prefix pm- to the tokens, like I did with wl-in:
`htmldef "wl-in" as " : ";
  althtmldef "wl-in" as " : ";
  latexdef "wl-in" as "\text{ : }";`

Reply:
"ElEqvRel" was my token as well, this is just a renaming in my Mathbox. 

I think you are completely right (i.e., we need a solution which makes possible parallel development), especially in cases when we already have a definition for the same concept in set.mm , i.e., when you explicitly want to change a definition of an existing concept. However, this is not the case at "EqvRel" or "CoElEqvRel": e.g. the concept of my definition
```metamath
df-eqvrel $a |- ( EqvRel R <-> ( RefRel R /\ SymRel R /\ TrRel R ) ) $.
```
, however conventional outside of set.mm, is not in the main set.mm , so we do not need a superfluously personalized token for this to avoid possible future cases. If, in the future, somebody finds a better definition for this concept, s/he may use our [convention](https://us.metamath.org/mpeuni/conventions.html) for his/her case:  "Alternative (ALTV) versions. If a theorem or definition is an alternate/variant of an existing theorem or definition, its label should have the same name with suffix ALTV. Such alternates are often temporary only, until it is decided which alternative should be used in the future." 

Our convention above is only about the ending of the theorem label, but I use it in stead of your "pm-*" suggestion in the token as well, like "*ALTV". For example, in case of the "equivalence relation with domain" concept, I think we should retire our present 
```metamath
df-er $a |- ( R Er A <-> ( Rel R /\ dom R = A /\ ( `' R u. ( R o. R ) ) C_ R ) ) $.
```
and use
```metamath
  dferALTV2 $p |- ( R ErALTV A <-> ( EqvRel R /\ ( dom R /. R ) = A ) ) $=
```
. If we are recommended to use "ALTV" in the label, I think we should try to use "ALTV" in the token as well and avoid our conventional "df-erALTV" label with an apparently different kind of personalized "R pm-Er A" token. I think the definition of equivalence relation in set.mm is a not so crowded space which would need more personalized tokens than the ALTV-solution I use.